.github/ISSUE_TEMPLATE/new_release_repository.md

```diff
@@ -7,8 +7,18 @@ title: 'Add new release repositories'
 * Name of the release team: 
 
 * Release repositories to add:
+  <!--
+  Before release repositories are created, repositories and packages should be submitted to the official rosdistro repository https://github.com/ros/rosdistro for naming review.
+  The ros/rosdistro source entry link must be included for each repository.
+  Guidelines for package naming are describe in REP-144: https://www.ros.org/reps/rep-0144.html
+
+  If there are any existing release repositories which should have their contents imported into the ros2-gbp organization, list and link to them here.
+  Leave the checkbox _unchecked_. The ros2-gbp administrator will check the box when they have completed the repository import.
+  -->
   * [FIRST_REPOSITORY](SOURCE URL)
```

Comment:
Given a PR to `rosdistro` that lists up required info e.g. source repo URL is a must, shouldn't a link to such PR suffice? If so, we can get rid of `[FIRST_REPOSITORY](SOURCE URL)` line, for simplification?

Reply:
This is a great point! When the template was first written the rosdistro PR was not yet a requirement. It's slightly "more work" for the maintainer to have to follow the the source url through the rosdistro PR but I would much rather do that work in order to simplify the template! Thanks for the feedback!

Reply:
Thanks again for the feedback, I've incorporated it into a new draft PR here https://github.com/ros2-gbp/ros2-gbp-github-org/pull/296 but I'm also considering a larger restructuring of the documentation for release repository creation.